routes/fcmToken.js

```diff
@@ -3,9 +3,10 @@ const router = express.Router();
 
 const authenticate = require("../middlewares/authenticate");
 const { fcmTokenController } = require("../controllers/fcmToken");
+const { fcmTokenValidator } = require("../middlewares/validators/fcmToken");
 
 // the route should be authenticated
-router.post("/", authenticate, fcmTokenController);
+router.post("/", fcmTokenValidator, authenticate, fcmTokenController);
```

Comment:
will it be a good idea to call authenticate before fcmTokenValidator so that only when a user is authenticated we call fcmTokenValidator

Reply:
updated